autoload/neomake/makers/ft/vue.vim

```diff
@@ -0,0 +1,21 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#vue#EnabledMakers()
```

Comment:
To make the linters happy, please add `abort` at the end of each `:function` line.